### PR TITLE
refactor: move default alert rules into the collection

### PR DESCRIPTION
**Pull request by murrant**
Also adds the wireless rules to the default set.

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: html/includes/print-alert-rules.php [hunk 1]
```diff
@@ -11,82 +11,23 @@
...
     require_once '../includes/alerts.inc.php';
+
     foreach ($default_rules as $add_rule) {
-        $add_rule['query'] = GenSQL($add_rule['rule']);
-        dbInsert($add_rule, 'alert_rules');
+        $insert = array(
+            'device_id' => -1,
+            'query'     => GenSQL($add_rule['rule']),
+            'severity'  => 'critical',
+            'extra'     => '{"mute":false,"count":"1","delay":"300"}',
```
Comment (laf) [inline html/includes/print-alert-rules.php]: Only one alert to be sent? I know it's user editable but should we consider -1 still for unlimited?

File: html/includes/print-alert-rules.php [hunk 2]
```diff
@@ -11,82 +11,23 @@
...
     require_once '../includes/alerts.inc.php';
+
     foreach ($default_rules as $add_rule) {
-        $add_rule['query'] = GenSQL($add_rule['rule']);
-        dbInsert($add_rule, 'alert_rules');
+        $insert = array(
+            'device_id' => -1,
+            'query'     => GenSQL($add_rule['rule']),
+            'severity'  => 'critical',
+            'extra'     => '{"mute":false,"count":"1","delay":"300","invert":false,"interval":300}',
```
Comment (laf) [inline html/includes/print-alert-rules.php]: My old comment got hidden:

Only one alert to be sent? I know it's user editable but should we consider -1 still for unlimited?

Comment (murrant) [inline html/includes/print-alert-rules.php]: This only affects when you click add default alerts.  If a user selects one from the collection, those default values are elsewhere. Defaults here we're copied exactly from the old code.

Comment (laf) [inline html/includes/print-alert-rules.php]: The old code had a mix of 1 and -1 based on the type of rule :)

Comment (murrant) [inline html/includes/print-alert-rules.php]: Oh, I see it now.  I'd say default to 1 and add extra to the rules that have -1...

**Pull request merged**